index.yaml

```diff
@@ -1,6 +1,28 @@
 apiVersion: v1
 entries:
   cluster-autoscaler-chart:
+  - apiVersion: v2
+    appVersion: 1.18.1
+    created: "2020-10-03T19:01:15.137204+01:00"
+    description: Scales Kubernetes worker nodes within autoscaling groups.
+    digest: 8b016745b3cb5ed2a3e7760b61d404b6debb2d5c252b0e524901bb8b89aef3b0
+    home: https://github.com/kubernetes/autoscaler
+    icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
+    maintainers:
+    - email: e.bailey@sportradar.com
+      name: yurrriq
+    - email: mgoodness@gmail.com
+      name: mgoodness
+    - email: guyjtempleton@googlemail.com
+      name: gjtempleton
+    - email: scott.crooks@gmail.com
+      name: sc250024
+    name: cluster-autoscaler-chart
+    sources:
+    - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
+    urls:
+    - https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-1.0.3/cluster-autoscaler-chart-1.0.4.tgz
```

Comment:
Do we not have an error here? Should it not be `cluster-autoscaler-chart-1.0.4`?

```
https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-1.0.3/cluster-autoscaler-chart-1.0.4.tgz
```

vs

```
https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-1.0.4/cluster-autoscaler-chart-1.0.4.tgz
```

Reply:
yes, this is wrong, the download failed = 

```
Error: failed to download "https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-1.0.3/cluster-autoscaler-chart-1.0.4.tgz" (hint: running `helm repo update` may help)
```

Reply:
I created this PR to fix it https://github.com/kubernetes/autoscaler/pull/3577